css-brochure-form/styles.css

```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+form {
+  margin: auto;
+  max-width: 800px;
```

Comment:
If you change this to something smaller (maybe like 30%), you shouldn't need to push all your content around with padding anymore.

Reply:
no padding this time!
![image](https://user-images.githubusercontent.com/97006645/152070577-87dc9423-e238-471c-9e8e-80772ea02615.png)